.github/workflows/sycl-precommit.yml

```diff
@@ -98,23 +105,16 @@ jobs:
             install_drivers: ${{ contains(needs.detect_changes.outputs.filters, 'drivers') }}
             extra_lit_opts: --param matrix-xmx8=True --param gpu-intel-dg2=True
             env: '{"LIT_FILTER":${{ needs.determine_arc_tests.outputs.arc_tests }} }'
-          - name: E2E tests with dev igc on Intel Arc A-Series Graphics
+
+          - name: E2E on Intel Arc A-Series Graphics
```

Comment:
Should the name be more specific on what targets are being tested, for example: OpenCL FPGA

Reply:
Probably, I'll have a look